broker-daemon/relayer/market-watcher.js

```diff
@@ -107,7 +110,7 @@ class MarketWatcher extends EventEmitter {
     this.logger.debug(`response type is ${response.type}`)
 
     if (RESPONSE_TYPES[response.type] === RESPONSE_TYPES.START_OF_EVENTS) {
-      this.migrate()
+      await this.migrate()
```

Comment:
This should *not* await, but should instead kick off migration. (Although I'm not sure it makes a difference)